src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+import * as React from 'react';
+
+import * as Data from '../data';
+
+const fetchSettings: RequestInit = {
+    credentials: "same-origin",
+    headers: {
+        ["X-Requested-With"]: "XMLHttpRequest"
+    },
+};
+
+class LoadingPanel extends React.Component<undefined, undefined> {
+    render() {
+        return <div className="progress">
+            <div className="progress-bar progress-bar-striped active" style={{ width: "100%" }}>
+                <span className="sr-only">Loading...</span>
+            </div>
+        </div>
+    }
+};
+
+class ErrorPanel extends React.Component<{ message: string }, undefined> {
+    render() {
+        return <div className="alert alert-danger">{this.props.message}</div>;
+    }
+};
+
+class UserIcon extends React.Component<{ user: Data.User }, undefined> {
+    render() {
+        return <a href={this.props.user.url}>
+            <img style={{ "margin": "0.1em" }} height="32" width="32" src={this.props.user.avatarUrl} title={this.props.user.login} alt={this.props.user.login} />
+        </a>;
+    }
+}
+
+class IssueUserView extends React.Component<{ issue: Data.Issue }, undefined> {
+    render() {
+        let issue = this.props.issue;
+        let issueLink = <span className="issue-link">
+            <a href={issue.url}>
+                #{issue.number}
+            </a>
+        </span>;
+
+        let style = { "whiteSpace": "nowrap" };
+        if (issue.assignees && issue.assignees[0] && issue.assignees[0].id != issue.author.id) {
```

Comment:
Another stylistic point, but this one's a but more important - always use strict equality/inequality operators (in this case, `!==` rather than `!=`) unless for some incredibly rare reason you actually want the crazy semantics of loose equality/inequality, and in that case it warrants a comment to explain why loose equality/inequality is the relevant behaviour.